Client/Ecosia/L10N/en.lproj/Ecosia.strings

```diff
@@ -153,8 +153,8 @@
 "Trees planted" = "Trees planted";
 "Before ..." = "Before ...";
 "After" = "After";
-"trees planted by the Ecosia community" = "trees planted by the Ecosia community";
-"Trees planted by the Ecosia community" = "Trees planted by the Ecosia community";
+"trees planted by Ecosia" = "trees planted by Ecosia";
+"Trees planted by Ecosia" = "Trees planted by Ecosia";
```

Comment:
Don't you need to update the place that uses them given the key updated?

Reply:
Of course I do! 🤣 
Too many back and forth with Firefox Upgrade already! Will update 🙏